### PR TITLE
Backend start

### PR DESCRIPTION
**Pull request by brunofoster03**
Created initial backend configuration with Node.JS and Express.JS. With app and database configuration, sequelize models and routes configuration

### CODE REVIEW

File: src/models/Extra.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const { DataTypes } = require('sequelize')
+
+// Create & export Extra model
+module.exports = (sequelize) => {
+  sequelize.define('Extra', {
```
Comment (manduinca) [inline src/models/Extra.js]: Maybe a better name can be: `TripAdditionalCharges`

File: src/database.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+const sequelize = new Sequelize(`postgres://${DB_USER}:${DB_PASSWORD}@${DB_HOST}:${DB_PORT}/${DB_NAME}`, {
+  logging: false,
+  native: false
+})
+
+// Imports models
+const EnterpriceModel = require('./models/Enterprice.js')
+const ExtraModel = require('./models/Extra.js')
+const ReserveModel = require('./models/Reserve.js')
+const RolModel = require('./models/Rol.js')
```
Comment (manduinca) [inline src/database.js]: Role?

File: src/database.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+const { DB_USER, DB_PASSWORD, DB_HOST, DB_PORT, DB_NAME } = process.env
+
+// Database sequelize connection
+const sequelize = new Sequelize(`postgres://${DB_USER}:${DB_PASSWORD}@${DB_HOST}:${DB_PORT}/${DB_NAME}`, {
+  logging: false,
+  native: false
+})
+
+// Imports models
+const EnterpriceModel = require('./models/Enterprice.js')
```
Comment (manduinca) [inline src/database.js]: Enterprise?

File: src/database.js
Comment (manduinca) [inline src/database.js:1]: Please review names. I think we are missing someone changes.

File: .vscode/settings.json
Comment (manduinca) [inline .vscode/settings.json:1]: Please don't push it to repository. You can add `.vscode` line to gitignore file.

**Review by manduinca — CHANGES_REQUESTED** [review]
I leaved some comments.

File: src/models/Rol.js
Comment (manduinca) [inline src/models/Rol.js:1]: Please don't forget this file. Role.

**Review by manduinca — CHANGES_REQUESTED** [review]
Please change rol to role.

**Review by martinalturria — DISMISSED** [review]

File: src/models/Rol.js
Comment (manduinca) [inline src/models/Rol.js:1]: I think you forget this file, please delete it.

**Review by manduinca — CHANGES_REQUESTED** [review]

**Review by manduinca — APPROVED** [review]
LGTM!

**Review by martinalturria — APPROVED** [review]
LGTM

**Pull request merged**